tests/src/Interop/PInvoke/Array/MarshalArrayAsField/LPArrayNative/MarshalArrayByValArrayNative.cpp

```diff
@@ -70,10 +70,64 @@ typedef struct  { BOOL		 arr[ARRAY_SIZE];		}	S_BOOLArray;
 helper function
 ----------------------------------------------------------------------------*/
 
+template<typename T>
+bool IsObjectEquals(T o1, T o2)
```

Comment:
This will only work for primitive types. The signature should be `(T& o1, T& o2)` at the very least.